test/UnitTest/accumulator.py

```diff
@@ -7,20 +7,18 @@
 myStatus = 'off'
 
 subId = []
-entityIdDict = {}
+entityIdDict = []
```

Comment:
Better to provide the variable name as it functionality. it can be modified in next PR.